app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string name="google_maps_key">AIzaSyBY3c7RFvwzs06juEg0nzIU7PVERTIV3U8</string>
+    <string name="facebook_app_id">Facebook</string>
```

Comment:
Why you added this String resources?

Reply:
During the downloading period of the project, Android Studio requested to add/update String resources.